arroyo/processing/strategies/__init__.py

```diff
@@ -22,4 +23,7 @@
     "ProcessingStrategyFactory",
     "Produce",
     "RunTaskInThreads",
+    "BatchStep",
+    "UnbatchStep",
+    "ValuesBatch",
```

Comment:
I don't think it's needed for most people - let's not export ValuesBatch from here?

Reply:
done